openapi/components/schemas/questionnaire.json

```diff
@@ -1,26 +1,25 @@
 {
   "title": "Questionnaire",
-  "additionalProperties": false,
-  "description": "The questionaire.",
+  "description": "The questionnaire.",
   "properties": {
     "id": {
       "description": "Unique, assigned identifier. Machine friendly.",
       "type": "string"
     },
     "meta": {
-      "$ref": "#/x-definitions/QuestionnaireMeta"
+      "$ref": "#/definitions/QuestionnaireMeta"
```

Comment:
we should use `x-definitions` as the openapi schema currently does allow `defintions`. 
![image](https://user-images.githubusercontent.com/13590797/80816033-0e24b800-8bcf-11ea-9565-a7bf7fd61040.png)